Pokecon/Program.cs

```diff
@@ -186,24 +289,36 @@ public static void Main(string[] args1)
             }
 
             set_location(args.location);
-
-            var access_token = login_ptc(args.username, args.password);
+            string access_token;
+            if (args.method == "ptc")

```

Comment:
it's actually case sensitive, because the method string is later used as a parameter in a post request to login.